gems/activerecord/6.0/activerecord.rbs

```diff
@@ -52,26 +52,26 @@ module ActiveRecord
     def self.enum: (Hash[Symbol, untyped]) -> void
 
     # callbacks
-    def self.after_commit: (*untyped) -> void
+    def self.after_commit: (*untyped) ?{ () -> untyped} -> void
     def self.after_create_commit: (*untyped) -> void | ...
     def self.after_update_commit: (*untyped) -> void | ...
     def self.after_destroy_commit: (*untyped) -> void | ...
     def self.after_save_commit: (*untyped) -> void | ...
```

Comment:
[Question]
The methods in lines 56-59 are defined in the `ActiveRecord::Transactions` module.
https://github.com/rails/rails/blob/4962e03683a6a16627d8f130bd6b767e79963918/activerecord/lib/active_record/transactions.rb#L231-L266

These methods also accept blocks, but should they be rewritten to type definitions similar to other callbacks?
Sorry, I am not sure about the `| ...` part, so I may be asking a question that is off the mark.

```suggestion
    def self.after_create_commit: (*untyped) ?{ () -> untyped} -> void
    def self.after_update_commit: (*untyped) ?{ () -> untyped} -> void
    def self.after_destroy_commit: (*untyped) ?{ () -> untyped} -> void
    def self.after_save_commit: (*untyped) ?{ () -> untyped} -> void
```

Reply:
> These methods also accept blocks, but should they be rewritten to type definitions similar to other callbacks?

The overloads are added since https://github.com/ruby/gem_rbs_collection/pull/138.

They work correctly, so I'll merge this PR without changes for after_create_commit. If you'd like to refactor them, feel free to do that. 

---

`...` is a syntax for overloading. It is not well described in the documentation, so I opened a PR to describe it. https://github.com/ruby/rbs/pull/1120